app/main/routes.py

```diff
@@ -21,7 +21,7 @@ def index():
 
 
 @bp.route("/upload", methods=["GET", "POST"])
-@login_required(return_app=SupportedApp.POST_AWARD_SUBMIT)
+@login_required(return_app=SupportedApp.POST_AWARD_SUBMIT, roles_required=["TF_MONITORING_RETURN_SUBMITTER"])
```

Comment:
Could `TF_MONITORING_RETURN_SUBMITTER` be defined as a constant and then referenced throughout the application

Reply:
OK have done this now in: 8717627